aws-rds-globalcluster/docs/README.md

```diff
@@ -58,7 +58,7 @@ _Required_: No
 
 _Type_: String
 
-_Update requires_: [Replacement](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-replacement)
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
```

Comment:
That's actually not accurate - major version upgrade does cause interruptions, all of the global cluster goes down for the duration of MVU. "Updates with Some Interruption" from the link looks like a more accurate representation of the change.

Reply:
Good point, however, the "Updates with Some Interruption" flag doesn't appear to be used anywhere in this repo... not sure about the purpose of it in the doc...